panic error when exeu ct lint-and-install

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
I use chart-testing v2.2.0 docker： quay.io/helmpack/chart-testing:v2.2.0 to test ct lint and install:
docker exec --interactive ctd "$@" ct lint-and-install --charts stable/mysql,stable/postgresql,stable/etcd-operator,stable/coredns --namespace kube-system

then I get the error:

$HELM_HOME has been configured at /root/.helm.
Not installing Tiller due to 'client-only' flag having been set
Happy Helming!
"incubator" has been added to your repositories
No requirements found in stable/mysql/charts.
Linting chart 'stable/mysql'
Validating /workdir/stable/mysql/Chart.yaml...
Validation success! 👍
Validating maintainers...
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/helm/chart-testing/pkg/tool.parseOutGitRepoDomain(0xc0000142dc, 0x3, 0xc00004e400, 0xc000010330)
	/go/src/github.com/helm/chart-testing/pkg/tool/account.go:44 +0x6f
github.com/helm/chart-testing/pkg/tool.AccountValidator.Validate(0xc0000142dc, 0x3, 0xc000014170, 0x9, 0x0, 0x0)
	/go/src/github.com/helm/chart-testing/pkg/tool/account.go:29 +0x4d
github.com/helm/chart-testing/pkg/chart.(*Testing).ValidateMaintainers(0xc000084180, 0xc0000da4c0, 0xc, 0xc00007f860, 0x15)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:565 +0x202
github.com/helm/chart-testing/pkg/chart.(*Testing).LintChart(0xc000084180, 0xc0000da4c0, 0xc, 0x0, 0x0, 0x0, 0x3, 0x0, 0x0, 0xc000178720)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:312 +0x534
github.com/helm/chart-testing/pkg/chart.(*Testing).LintAndInstallChart(0xc000084180, 0xc0000da4c0, 0xc, 0x0, 0x0, 0x0, 0xc0000da4c0, 0xc, 0x0, 0x0)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:389 +0x67
github.com/helm/chart-testing/pkg/chart.(*Testing).LintAndInstallChart-fm(0xc0000da4c0, 0xc, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:257 +0x63
github.com/helm/chart-testing/pkg/chart.(*Testing).processCharts(0xc000084180, 0xc0000e7828, 0xc000097832, 0x922d80, 0xc3c4e0, 0x922da0, 0xc3c4e0)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:232 +0x6df
github.com/helm/chart-testing/pkg/chart.(*Testing).LintAndInstallCharts(0xc000084180, 0x3, 0xc000097298, 0x6, 0x0, 0x0)
	/go/src/github.com/helm/chart-testing/pkg/chart/chart.go:257 +0x4e
github.com/helm/chart-testing/app/cmd.lintAndInstall(0xc00014d180, 0xc0000aae00, 0x0, 0x4)
	/go/src/github.com/helm/chart-testing/app/cmd/lintAndInstall.go:53 +0x20c
github.com/helm/chart-testing/vendor/github.com/spf13/cobra.(*Command).execute(0xc00014d180, 0xc0000aadc0, 0x4, 0x4, 0xc00014d180, 0xc0000aadc0)
	/go/src/github.com/helm/chart-testing/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/helm/chart-testing/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc00014ca00, 0xc000098120, 0xc00014d680, 0xc00014d400)
	/go/src/github.com/helm/chart-testing/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/helm/chart-testing/vendor/github.com/spf13/cobra.(*Command).Execute(0xc00014ca00, 0x405690, 0xc000076058)
	/go/src/github.com/helm/chart-testing/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/helm/chart-testing/app/cmd.Execute()
	/go/src/github.com/helm/chart-testing/app/cmd/root.go:57 +0x27
main.main()
	/go/src/github.com/helm/chart-testing/app/main.go:22 +0x20
+ docker container rm --force ctd

**Version of Helm and Kubernetes**:

helm version: v2.12.2 (including in chart-testing image)
kubectl version: v1.14.0 and v1.12.7, both failed with the above error msg
ct version: v2.2.0

--- a/pkg/tool/account.go
+++ b/pkg/tool/account.go
@@ -16,30 +16,37 @@ package tool
 
 import (
 	"fmt"
-	"github.com/pkg/errors"
 	"net/http"
 	"regexp"
+
+	"github.com/pkg/errors"
 )
 
 type AccountValidator struct{}
 
 var repoDomainPattern = regexp.MustCompile("(?:https://|git@)([^/:]+)")
 
-func (v AccountValidator) Validate(repoUrl string, account string) error {
-	domain := parseOutGitRepoDomain(repoUrl)
+func (v AccountValidator) Validate(repoURL string, account string) error {
+	domain, err := parseOutGitRepoDomain(repoURL)
+	if err != nil {
+		return err
+	}
 	url := fmt.Sprintf("https://%s/%s", domain, account)
 	response, err := http.Head(url)
 	if err != nil {
 		return errors.Wrap(err, "Error validating maintainers")
 	}
 	if response.StatusCode != 200 {
-		return errors.New(fmt.Sprintf("Error validating maintainer '%s': %s", account, response.Status))
+		return fmt.Errorf("Error validating maintainer '%s': %s", account, response.Status)
 	}
 	return nil
 }
 
-func parseOutGitRepoDomain(repoUrl string) string {
+func parseOutGitRepoDomain(repoURL string) (string, error) {
 	// This works for GitHub, Bitbucket, and Gitlab
-	submatch := repoDomainPattern.FindStringSubmatch(repoUrl)
-	return submatch[1]
+	submatch := repoDomainPattern.FindStringSubmatch(repoURL)
+	if submatch == nil || len(submatch) < 2 {
+		return "", fmt.Errorf("Could not parse git repository domain for '%s'", repoURL)
+	}
+	return submatch[1], nil
 }

--- a/pkg/tool/account_test.go
+++ b/pkg/tool/account_test.go
@@ -1,8 +1,10 @@
 package tool
 
 import (
-	"github.com/stretchr/testify/assert"
+	"fmt"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestParseOutGitDomain(t *testing.T) {
@@ -10,18 +12,21 @@ func TestParseOutGitDomain(t *testing.T) {
 		name     string
 		repoUrl  string
 		expected string
+		err      error
 	}{
-		{"GitHub SSH", "git@github.com:foo/bar", "github.com"},
-		{"GitHub HTTPS", "https://github.com/foo/bar", "github.com"},
-		{"Gitlab SSH", "git@gitlab.com:foo/bar", "gitlab.com"},
-		{"Gitlab HTTPS", "https://gitlab.com/foo/bar", "gitlab.com"},
-		{"Bitbucket SSH", "git@bitbucket.com:foo/bar", "bitbucket.com"},
-		{"Bitbucket HTTPS", "https://bitbucket.com/foo/bar", "bitbucket.com"},
+		{"GitHub SSH", "git@github.com:foo/bar", "github.com", nil},
+		{"GitHub HTTPS", "https://github.com/foo/bar", "github.com", nil},
+		{"Gitlab SSH", "git@gitlab.com:foo/bar", "gitlab.com", nil},
+		{"Gitlab HTTPS", "https://gitlab.com/foo/bar", "gitlab.com", nil},
+		{"Bitbucket SSH", "git@bitbucket.com:foo/bar", "bitbucket.com", nil},
+		{"Bitbucket HTTPS", "https://bitbucket.com/foo/bar", "bitbucket.com", nil},
+		{"Invalid", "foo/bar", "", fmt.Errorf("Could not parse git repository domain for 'foo/bar'")},
 	}
 
 	for _, testData := range testDataSlice {
 		t.Run(testData.name, func(t *testing.T) {
-			actual := parseOutGitRepoDomain(testData.repoUrl)
+			actual, err := parseOutGitRepoDomain(testData.repoUrl)
+			assert.Equal(t, err, testData.err)
 			assert.Equal(t, testData.expected, actual)
 		})
 	}

